test/integration/utils.go

```diff
@@ -143,3 +154,102 @@ func (e *eqWrapper) Used(used v1.ResourceList) *eqWrapper {
 func (e *eqWrapper) Obj() *v1alpha1.ElasticQuota {
 	return e.ElasticQuota
 }
+
+type testContext struct {
```

Comment:
any particular reason to compose one instead of reusing the upstream TestContext?